### PR TITLE
Add clientId to fetch event (#131)

### PR DESCRIPTION
**Pull request by jRichardeau**
From #131 add possibility to pass a _clientId_ through the fetch trigger so that it will be available in the fetch event listeners.

### CODE REVIEW

File: packages/service-worker-mock/models/FetchEvent.js [hunk 1]
```diff
@@ -2,19 +2,28 @@ const ExtendableEvent = require('./ExtendableEvent');
...
-  }
+   constructor(type, init) {
+      super();
+      this.type = type;
+      this.isReload = init.isReload || false;
+      this.clientId = null;
+
+      let request = init.request;
+
+      if (typeof request === "object" && request.request) {
```
Comment (zackargyle) [inline packages/service-worker-mock/models/FetchEvent.js]: According to the spec, init.request should only be a Request or a string. Can you move this logic into eventHandler.js instead?

Comment (jRichardeau) [reply packages/service-worker-mock/models/FetchEvent.js]: Ok I misunderstand what you wanted

**Comment by jRichardeau** [comment]
Thank you for accepting !
When this could be available on npm ?
Thanks

**Comment by zackargyle** [comment]
@jRichardeau It is now available as 2.0.5. Thanks again!

**Comment by jRichardeau** [comment]
Oh, I wanted this fix in 1.x that's why I did it in the _publish-1.10.0_ branch.
2.x version is not working for my tests so is it possible to create a new 1.x version with this ?
Thank you

**Comment by jRichardeau** [comment]
I also think you should revert this version (2.0.5) and the merge to master as it breaks the 2.x API because I did it from the 1.x version.

**Comment by jRichardeau** [comment]
@zackargyle sorry for annoyance but is it possible to merge this feature in the 1.x version ? Moreover I think it should not be merged in the 2.x version.
Thank you

**Comment by jRichardeau** [comment]
Hey any news on this merge ?
Thanks

**Comment by jRichardeau** [comment]
@zackargyle hey, can you look at my previous messages please ? Could you merge this on the 1.x version ?

**Pull request merged**